doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -913,6 +923,21 @@ Note that if a custom payload is used for the notification (the field `payload`
 corresponding subscription), then a value of `custom` is used for the `Ngsiv2-AttrsFormat` header
 in the notification.
 
+## Special metadata in notifications
```

Comment:
Maybe. But I move that discussion outside this PR (it also affects to attributes, such as dataCreated, which call be named "System attribute").